### PR TITLE
[d16-7][xcode11.6] Bump to final version of Xcode 11.6

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/MetalPerformanceShaders/MPSDefs.cs [hunk 1]
```diff
@@ -74,14 +75,30 @@ public enum MPSDataType : uint { // uint32_t
 		Unorm8 = NormalizedBit | 8,
 	}
 
-	[iOS (10,0)][TV (10,0)][Mac (10, 13)]
+	[Flags]
+	[Native]
+	public enum MPSAliasingStrategy : ulong
```
Comment (spouliot) [inline src/MetalPerformanceShaders/MPSDefs.cs:80]: I realize it's a backport (I seem to have missed the PR) but this, and others, seems to be missing availability attributes

File: src/MetalPerformanceShaders/MPSDefs.cs [hunk 2]
```diff
@@ -164,6 +188,15 @@ public enum MPSMatrixDecompositionStatus {
 		NonPositiveDefinite = -3,
 	}
 
+	[Introduced (PlatformName.MacCatalyst, 13, 0)]
```
Comment (spouliot) [inline src/MetalPerformanceShaders/MPSDefs.cs:191]: missing other attributes ?

File: src/MetalPerformanceShaders/MPSStateBatch.cs [hunk 3]
```diff
@@ -43,11 +44,13 @@ public static void Synchronize (NSArray<MPSState> stateBatch, IMTLCommandBuffer
 			MPSStateBatchSynchronize (stateBatch.Handle, commandBuffer.Handle);
 		}
 
+		[Introduced (PlatformName.MacCatalyst, 13, 0)]
```
Comment (spouliot) [inline src/MetalPerformanceShaders/MPSStateBatch.cs:47]: already added on the type itself

File: src/MetalPerformanceShaders/MPSStateBatch.cs [hunk 4]
```diff
@@ -43,11 +44,13 @@ public static void Synchronize (NSArray<MPSState> stateBatch, IMTLCommandBuffer
...
 		[iOS (12,0), TV (12,0), Mac (10,14)]
 		[DllImport (Constants.MetalPerformanceShadersLibrary)]
 		static extern nuint MPSStateBatchResourceSize (IntPtr batch);
 
 		// Using 'NSArray<MPSState>' instead of `MPSState[]` because array 'Handle' matters.
+		[Introduced (PlatformName.MacCatalyst, 13, 0)]
```
Comment (spouliot) [inline src/MetalPerformanceShaders/MPSStateBatch.cs:53]: same

**Comment by dalexsoto** [comment]
As our conversation those availability attributes will be fixed in Xcode 12

**Comment by dalexsoto** [comment]
Build

**Pull request merged**